src/service.js

```diff
@@ -67,7 +67,7 @@ export class Disposable {
 export function getExistingServiceForDocInEmbedScope(element, id) {
   // TODO(#22733): completely remove this method once ampdoc-fie launches.
   // Resolve via the element's ampdoc.
-  return getServiceForDocOrNullInternal(element, id);
```

Comment:
can you explain this rename? I don't have much context on what internal means. Does it mean a non-amp element (node within an amp elem). I'm guessing the addition of `AmpDoc` as a potential param is an optimization?

Reply:
I initially renamed it to `internal` b/c it was not used outside of this file. But now it is. Also, per `TODO` there, the `getExistingServiceForDocInEmbedScope` will be replaced with this call everywhere. It's done in the https://github.com/ampproject/amphtml/issues/22733. So there's a slight race between two things I'm working on :)